client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -53,61 +87,198 @@ function HelpModal({
+        closeHelpModal();
+        setShowHelpForm(false);
+        resetFormValues();
+      }}
+      show={isOpen}
+    >
       <Modal.Header className='help-modal-header fcc-modal' closeButton={true}>
         <Modal.Title className='text-center'>
           {t('buttons.ask-for-help')}
         </Modal.Title>
       </Modal.Header>
       <Modal.Body className='help-modal-body text-center'>
-        <h3 className='help-modal-heading'>
-          <Trans i18nKey='learn.tried-rsa'>
-            <a
-              href={RSA}
-              rel='noopener noreferrer'
-              target='_blank'
-              title={t('learn.rsa')}
+        {showHelpForm ? (
+          <form
+            onSubmit={event => {
+              event.preventDefault();
+              if (!canSubmitForm()) return;
+              setShowHelpForm(false);
+              resetFormValues();
+              createQuestion();
+              closeHelpModal();
+            }}
+          >
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'column'
+              }}
             >
-              placeholder
-            </a>
-          </Trans>
-        </h3>
-        <div className='alert alert-danger'>
-          <FontAwesomeIcon icon={faExclamationCircle} />
-          <p>
-            <Trans i18nKey='learn.rsa-forum'>
-              <a
+              <CheckboxHelpModal
+                name='read-search-ask-checkbox'
+                i18nkey={t('learn.read-search-ask-checkbox')}
+                onChange={event =>
+                  handleCheckboxChange(event, setReadSearchCheckbox)
+                }
+                value={readSearchCheckbox}
+                href={RSA}
+                t={t}
+              />
+              <CheckboxHelpModal
+                name='similar-questions-checkbox'
+                i18nkey={t('learn.similar-questions-checkbox')}
+                onChange={event =>
+                  handleCheckboxChange(event, setSimilarQuestionsCheckbox)
+                }
+                value={similarQuestionsCheckbox}
                 href={generateSearchLink(challengeTitle, challengeBlock)}
-                rel='noopener noreferrer'
-                target='_blank'
-              >
-                placeholder
-              </a>
-              placeholder
-            </Trans>
-          </p>
-        </div>
-        <Button
-          block={true}
-          size='large'
-          variant='primary'
-          onClick={createQuestion}
-        >
-          {t('buttons.create-post')}
-        </Button>
-        <Spacer size='xxSmall' />
-        <Button
-          block={true}
-          size='large'
-          variant='primary'
-          onClick={closeHelpModal}
-        >
-          {t('buttons.cancel')}
-        </Button>
+                t={t}
+              />
+              <label htmlFor='description'>
+                {t('forum-help.whats-happening')}
+              </label>
```

Comment:
```suggestion
              <label htmlFor='help-modal-form-description'>
                {t('forum-help.whats-happening')}
                <span className='sr-only'>50 character minimum</span>
              </label>
```

Of course this needs to be translated, I'm just too lazy to do that.